pkg/vmprovider/providers/vsphere2/network/network.go

```diff
@@ -557,9 +557,8 @@ func createVPCNetworkInterface(
 	}
 
 	switch interfaceSpec.Network.Kind {
-	case "SubnetSet":
-		vpcSubnetPort.Spec.SubnetSet = networkName
-	case "":
```

Comment:
Can you drop this part of the commit? Because of some other recent changes I think we can handle this better with a small change in the mutation webhook.